pubsub.go

```diff
@@ -28,6 +28,10 @@ const DefaultMaxMessageSize = 1 << 20
 
 var (
 	TimeCacheDuration = 120 * time.Second
+
+	// ErrSubscriptionCancelled may be returned when a subscription Next() is called after the
+	// subscription has been cancelled.
+	ErrSubscriptionCancelled = errors.New("subscription cancelled by calling sub.Cancel()")
```

Comment:
let's call the error message just "subscription cancelled"; no need to perpetuate a bad error message.

Reply:
```suggestion
	ErrSubscriptionCancelled = errors.New("subscription cancelled")
```